src/components/structures/UserMenu.tsx

```diff
@@ -234,12 +234,12 @@ export default class UserMenu extends React.Component<IProps, IState> {
         >
             <div className="mx_UserMenu_contextMenu_header">
                 <div className="mx_UserMenu_contextMenu_name">
-                        <span className="mx_UserMenu_contextMenu_displayName">
-                            {OwnProfileStore.instance.displayName}
-                        </span>
+                    <span className="mx_UserMenu_contextMenu_displayName">
+                        {OwnProfileStore.instance.displayName}
+                    </span>
                     <span className="mx_UserMenu_contextMenu_userId">
-                            {MatrixClientPeg.get().getUserId()}
-                        </span>
+                        {MatrixClientPeg.get().getUserId()}
+                    </span>
```

Comment:
I fixed this in https://github.com/matrix-org/matrix-react-sdk/pull/5161 by making the most complicated merge conflict on the planet btw.